### PR TITLE
Rewrite plugin mechanics a bit

### LINKED ISSUE #21: Convert propertyWordDelimiters to a string
Char array is not a convenient way to define this parameter. Plugin should be able to separate it to a char array internally

### PR DESCRIPTION
**Pull request by eirnym**
* Split plugin configuration and library configuration
* Combine task for Android and for plain java
* Rename task
* Upgrade jsonschema2pojo library to 1.1.0

closes #21

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,9 +31,15 @@ gradlePlugin {
         }
     }
 }
+wrapper {
+    gradleVersion = '6.8.3'
```
Comment (hannesa2) [inline build.gradle:35]: Sorry, I've too less knowledge what's the purpose of if.
Why do you need it ?

File: src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy [hunk 2]
```diff
@@ -15,23 +15,38 @@
...
 import org.gradle.api.GradleException
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPluginConvention
 
 /**
  * Registers the plugin's tasks.
  *
  * @author Ben Manes (ben.manes@gmail.com)
+ * @author Arseny Nasokin (eirnym@gmail.com)
```
Comment (hannesa2) [inline src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy:27]: I would remove authors complete. When you add one ? It makes only maintenance, or a wrong history.
You see it in git history. But this is just my piont of view

**Review by hannesa2 — APPROVED** [review]
I can't really review it, because I've a weak groovy knowledge.
Best would be good samples/test, where CI at least check for a proper working plugin.
Eg the Android sample

File: src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy (hunk 2 above)
Comment (eirnym) [reply src/main/groovy/com/github/eirnym/js2p/JsonSchemaPlugin.groovy:27]: makes sense

File: build.gradle (hunk 1 above)
Comment (eirnym) [reply build.gradle:35]: Tried to add a gradle sources to play with

**Comment by eirnym** [comment]
> I can't really review it, because I've a weak groovy knowledge.
> Best would be good samples/test, where CI at least check for a proper working plugin.
> Eg the Android sample

there's not much groovy voodoo's going here 

basically there's only one place for a disaster left :) and targetDir is not working as well

I'll fix this disaster within android sample (rebased version is building locally)

File: build.gradle (hunk 1 above)
Comment (hannesa2) [reply build.gradle]: With my current knowledge, it's just noise

**Pull request merged**